app/models/translation.rb

```diff
@@ -76,6 +77,21 @@ def manifest_translated_phrases
     @manifest_translated_phrases ||= download_translated_phrases("name_description.xml")
   end
 
+  def push_published_to_s3
+    return if is_published
+
+    if resource.uses_onesky?
+      ActiveRecord::Base.transaction do
+        phrases = manifest_translated_phrases
+        name_desc_onesky(phrases)
+        create_translated_attributes(phrases)
+      end
+    end
+
+    Package.new(self).push_to_s3
+    update(is_published: true)
```

Comment:
let's also clear any `publishing_errors` here since we have just finished successfully publishing the translation.